composer/helpers/v1/composer.json

```diff
@@ -1,4 +1,7 @@
 {
+    "name": "dependabot/composer-v1-helper",
+    "description": "A helper package for Dependabot to perform updates using Composer",
+    "license": "The Prosperity Public License 2.0.0",
```

Comment:
PHPStan will still complain about this, since it's not a SPDX identifier.

I looked into submitting this license for inclusion into SPDX index, but after reading through https://github.com/spdx/license-list-XML/pull/960 I decided that wasn't worth pursuing further.

 I could have used `"proprietary"`, but decided to just ignore that warning for now.